js/copy-service/Substitutions/Substitutions.ts

```diff
@@ -55,7 +57,8 @@ class Substitutions {
    * @param {string} key Path to substitution on the substitutions object.
    * @returns {*} The function from the substitutions.
    */
-  getFunction(key) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  getFunction(key: string): string | (() => any) {
```

Comment:
We should fix this method on line 65 so it only returns type `Function`:

```js 
if (_.isUndefined(value)) {
  this._handleMissing(key);
  return;
}
```

Reply:
😩. Makes sense for consuming functions to handle `undefined` when this cannot find a function rather than an empty string.